client/app/dbaas/logs/logs-constants.js

```diff
@@ -324,7 +324,7 @@ angular.module('managerApp').constant('LogsConstants', {
       Twitter: {
         name: 'Twitter',
         input: 'twitter {\n\tconsumer_key => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"\n\tconsumer_secret => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx"\n\toauth_token => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"\n\toauth_token_secret => "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"\n\tkeywords => [ "playstation","Nintendo","xbox"]\n\ttype => "tweet"\n\tfull_tweet => true\n}',
```

Comment:
Why do we have the `playstation`, `Nintendo` and `xbox` keywords there ?

Reply:
It's related to the content of this documentation : https://docs.ovh.com/fr/logs-data-platform/twitter/ which analyzes Twitter trends around these 3 subjects

Reply:
of course, no problem at all  :-).